microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -153,7 +153,15 @@ To rename your new Snippet with a custom name, navigate to [Rename Snippets](#re
 
 1.  [Open the Snippets tab](#open-the-snippets-tab).  
 1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Remove**.  
```

Comment:
If you want to do a word-change throughout the entire article as part of this PR: change some instances of 'choose' to 'select', per Microsoft Writing Style Guide.  Public version of important page "Describing interactions with UI": https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui